gemini/master.py

```diff
@@ -32,25 +33,26 @@ def info() -> str:
 @app.route('/v1/nodes/join', methods=['POST'])
 def join() -> str:
     node_id = generate_node_id()
-    token = generate_node_token()
+    password = generate_node_password()
 
     value = {
         'node_id': node_id,
         'expires': 5000,
     }
 
     c.kv.put('nodes/' + node_id, json.dumps(value))
-    c.kv.put('nodes/tokens/' + token, node_id)
+    c.kv.put('nodes/' + password, node_id)
```

Comment:
We want to include the password as part of the value that is written to the `nodes/<node_id>` key. So that when a node authenticates with the cluster, it will give us the node ID and password, and we can query the node ID and see if the password is correct